static/templates/actors/familiar-sheet.hbs

```diff
@@ -210,32 +210,7 @@
                 </div>
                 <div class="section-body">
                     <ol class="effects-list item-list">
-                        {{#each items as |item idx|}}
-                            {{#if (eq item.type "effect")}}
-                                {{#if (or (not item.system.unidentified) @root.user.isGM)}}
-                                    <li class="item effects {{#if item.system.unidentified}}unidentified{{/if}}" data-item-id="{{item._id}}">
-                                        <div class="item-name">
-                                            <div class="item-image" style="background-image: url({{item.img}})">
-                                                <i class="fas fa-comment-alt"></i>
-                                            </div>
-                                            <h4>{{item.name}}</h4>
-                                        </div>
-                                        {{#if @root.options.editable}}
-                                            <div class="item-controls">
-                                                {{#if @root.user.isGM}}
-                                                    <a class="item-control" data-action="effect-toggle-unidentified" title="{{localize "PF2E.ToggleEffectUnidentified"}}">
-                                                        <i class="fas fa-fw fa-eye"></i>
-                                                    </a>
-                                                {{/if}}
-                                                <a class="item-control item-edit" title="{{localize "PF2E.EditItemTitle"}}"><i class="fas fa-edit"></i></a>
-                                                <a class="item-control item-delete" title="{{localize "PF2E.DeleteItemTitle"}}"><i class="fas fa-trash"></i></a>
-                                            </div>
-                                        {{/if}}
-                                        <div class="item-summary"></div>
-                                    </li>
-                                {{/if}}
-                            {{/if}}
-                        {{/each}}
+                        {{> systems/pf2e/templates/actors/partials/conditions.hbs conditions=document.itemTypes.effect}}
```

Comment:
Could we have a standard of quoting paths like this? The only thing that keeps it from getting mixed up with variables is it parsing like a path. I don't know that I want to trust handlebars to not fuck that up.

Reply:
I suspect its actually a unique feature of {{> judging by this:
https://handlebarsjs.com/guide/partials.html#basic-partials

unless of course those are all variables in the global context somehow, which sounds pretty unsafe.

But sure, I'll update them and a bunch of other ones.